conf/messages

```diff
@@ -2978,16 +2978,10 @@ supporting-evidence.expired.file-label=Document {0}
 supporting-evidence.expired.deleted=Deleted
 supporting-evidence.expired.button.tryAgain=Try again
 
-supporting-evidence.check-upscan-status.title=Supporting evidence file upload status
-supporting-evidence.check-upscan-status.caption=Supporting evidence
-supporting-evidence.check-upscan-status.in-progress.help-text=There is a delay in scanning your file. You can check the progress by clicking the button below.
-supporting-evidence.check-upscan-status.failed.help-text=Your supporting evidence file has failed its scan. You can try again by uploading another file.
-supporting-evidence.check-upscan-status.passed.help-text=Your supporting evidence file has been successfully saved and scanned. We will save it for 5 days or until you submit your return when it will be attached to your submission.
-supporting-evidence.check-upscan-status.passed=Passed
-supporting-evidence.check-upscan-status.failed=Failed
-supporting-evidence.check-upscan-status.uploaded=Scan in progress
-supporting-evidence.check-upscan-status.button.checkAgain=Check status and continue
-supporting-evidence.check-upscan-status.button.uploadAnotherFile=Upload another file
+supporting-evidence.scan-progress.title=Scan in progress
+supporting-evidence.scan-progress.caption=Supporting evidence
+supporting-evidence.scan-progress.p1=You''ll need to wait a few seconds
```

Comment:
we should use curly quotes instead of straight quotes. If you look around the messages file you'll find one to copy and paste, otherwise you can copy and paste from here: https://smartquotesforsmartpeople.com/

Reply:
fixed